src/clr-addons/quick-list/add-option.component.scss

```diff
@@ -0,0 +1,7 @@
+.allowed {
```

Comment:
set these css rules always but with the :not(disabled) selector and different class name